tiproxy/tiproxy-command-line-flags.md

```diff
@@ -188,14 +196,42 @@ tiproxyctl traffic replay --host 10.0.1.10 --port 3080 --username="u1" --passwor
 
 `tiproxyctl traffic show` 用于显示历史的捕获和回放任务。
 
-输出中的 `status` 字段表示任务的状态，它有下列值：
+输出中的 `status` 字段表示任务的状态，其可能的值包括：
 
-- `done`: 代表任务正常运行完成。
-- `canceled`: 代表任务被取消，`error` 字段描述了被取消的原因。
-- `running`: 代表任务正在运行，`progress` 字段描述了运行的进度。
+- `done`：任务正常完成。
+- `canceled`：任务被取消，查看 `error` 字段了解原因。
+- `running`：任务正在运行，查看 `progress` 字段了解进度。
 
 输出示例：
 
 ```json
-[{"type":"capture","start_time":"2024-09-01T14:30:40.99096+08:00","end_time":"2024-09-01T16:30:40.99096+08:00","duration":"2h","output":"/tmp/triffic","progress":"100%","status":"done"},{"type":"capture","start_time":"2024-09-02T18:30:40.99096+08:00","end_time":"2024-09-02T19:00:40.99096+08:00","duration":"2h","output":"/tmp/triffic","progress":"25%","status":"canceled","error":"canceled manually"},{"type":"capture","start_time":"2024-09-03T13:31:40.99096+08:00","duration":"2h","output":"/tmp/triffic","progress":"45%","status":"running"}]
+[
+  {
+    "type": "capture",
+    "start_time": "2024-09-01T14:30:40.99096+08:00",
+    "end_time": "2024-09-01T16:30:40.99096+08:00",
+    "duration": "2h",
+    "output": "/tmp/traffic",
```

Comment:
The directory `/tmp/triffic` appears to contain a typo. Did you mean `/tmp/traffic`?